src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/table_view.rs

```diff
@@ -0,0 +1,198 @@
+//! Table view implementation. Runs through JSON file creating a table view.
+
+use crate::data;
+use crate::component::visualization;
+
+/// Get table view stylesheet from a CSS file.
+fn style() -> String {
+    let _light = format!("<style>{}</style>", include_str!("table_view/style_light.css"));
+    let dark   = format!("<style>{}</style>", include_str!("table_view/style_dark.css"));
+    dark
+}
+
+/// Return a `JavaScript` Table view visualization.
+pub fn table_view_visualization() -> visualization::java_script::FallibleDefinition {
+    let source_beg = r#"
```

Comment:
The whole source should be a js file and should be included here with include_str macro.